src/main/include/Constants.h

```diff
@@ -304,6 +309,19 @@ constexpr double kWristSetIZone = 0.01;
 constexpr double kWristSetFF = 0.000015;
 }  // namespace ArmConstants
 
+// TODO: CREATE ACTUAL ROBOT VALUES FOR THESE
+namespace VisionConstants {
+    static constexpr std::string_view kCameraName{"PhotonVision"};
+    static const frc::Transform3d kRobotToCam{
+        frc::Translation3d{0.5_m, 0.0_m, 0.5_m},
```

Comment:
Might want to wrap in `ifdef` since the bots differ in LL placement

Reply:
yes